.bundle/config

```diff
@@ -1,2 +0,0 @@
----
-BUNDLE_FROZEN: "1"
```

Comment:
This is a removed file. It got added in this PR as I cherry-picked https://github.com/chef/chef/pull/5886/commits/2ff65b4f07617dbb1255b7b8f4795d69a35dcb39

@thommay , does it still affect the PR even though it's showing as a removed file?